tests/suites/test_suite_mpi.function

```diff
@@ -763,6 +763,316 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_core_cond_assign( data_t * input_X,
+                           data_t * input_Y,
+                           int input_len )
+{
+    mbedtls_mpi_uint *X = NULL;
+    mbedtls_mpi_uint *Y = NULL;
+    size_t limbs_X = CHARS_TO_LIMBS( input_X->len );
+    size_t limbs_Y = CHARS_TO_LIMBS( input_Y->len );
+    size_t limbs = limbs_X;
+    size_t copy_limbs = CHARS_TO_LIMBS( input_len );
+    size_t len = limbs * sizeof( mbedtls_mpi_uint );
```

Comment:
(minor; not worth changing at this point) The units of `len` are only clear when reading this line, but not in the body of the code; something like `bytes` would be clearer